docs/get-started/venv-tutorial.md

```diff
@@ -1,11 +1,16 @@
-## Getting started with venv & Planet SDK
+---
+title: Get started with venv & the Planet SDK for Python
+---
 
-There are lots of great resources online about virtual environments for python.
-This tutorial is aimed at users of the Planet SDK who have little or no experience
-with virtual environments but want to be early users of v2 (and still use
-their v1 when testing).
+This tutorial is aimed at Planet SDK for Python users who have little or no experience with virtual environments, especially those who want to be early users of v2 of the Planet SDK for Python (and still use v1 while evaluating v2).
 
-### Conda vs venv
+A virtual environment is a space you create to pull in the resources you need for a specific project. In Python, virtual environments are used to install tools that work well together in their own isolated area. While you may have Python installed already, this overview focuses on installing a version of Python in a way that ensures you have the right virtual environment for using this Planet SDK. 
+
+Using the default Python installation without a virtual environment is not recommended, because it’s so easy to get a combination of tools installed that don’t work well with each other. The great thing about installing a virtual environment is that if you get a combination of tools that don’t work well together, you can “throw away” that virtual environment and create a new one with a few lines in the terminal.
+
+So while it may feel awkward at first, spinning up virtual environments will be your go-to setup step for trying out a new idea, testing a routine, or repeating a process at scale.
```

Comment:
I wouldn't presume to know the user's experience here. And suggesting it is awkward may subconsciously push them into feeling it IS awkward, where they were pretty fine with it before the suggestion. I like the positive spin: tell them why it is helpful and tell them how it can be done the simplist way possible. And maybe give them a ref to find out more.